server/vertx/src/main/scala/sttp/tapir/server/vertx/VertxZioServerOptions.scala

```diff
@@ -26,7 +26,7 @@ object VertxZioServerOptions {
     CustomInterceptors(
       createOptions = (ci: CustomInterceptors[RIO[R, *], VertxZioServerOptions[RIO[R, *]]]) =>
         VertxZioServerOptions(
-          Defaults.createTempFile().getParentFile.getAbsoluteFile,
+          new java.io.File(System.getProperty("java.io.tmpdir")).getAbsoluteFile,
```

Comment:
I noted that new temp file is created each time when server is started. It seems that it is equivalent code to get temp dir.

Reply:
makes sense, thanks :)

Reply:
actually the same code is present in cats & future interpreters, so maybe we could make `ew java.io.File(System.getProperty("java.io.tmpdir"))` an internal utility function and simply use it in these places

Reply:
OK. I moved this line inside function `VertxServerOptions.uploadDirectory()`